test/fixtures/users.yml

```diff
@@ -28,3 +28,10 @@ bender:
   name: Bender Bot
   bot_token: <%= User.generate_bot_token %>
   role: bot
+
+banner:
```

Comment:
This is a bit confusing, I'd assume that banner is a person doing the banning, not the person getting banned.
Did you consider naming the fixture just bruce? or bruce_banner?